src/main/java/dev/andrylat/vomelianchuk/CardNumberOperations/enums/Errors.java

```diff
@@ -0,0 +1,17 @@
+package dev.andrylat.vomelianchuk.CardNumberOperations.enums;
+
+public enum Errors {
+    ERROR_LENGTH_CARD("Length should be 16 symbols"),
```

Comment:
We are in the class "Errors" and it is default that all enums are error. You don't need to duplicate it.

Reply:
Fixed here: 8741778